mitodl/ocw-studio
# Update docker-compose to work with Apple Silicon

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/1568.

#### What's this PR do?

All of the `docker-compose` overrides were merged into the main `docker-compose` file. This allows for an override file to be used specifically for anyone running OCW Studio locally on Apple Silicon.

#### How should this be manually tested?

If running on an Intel machine, run `docker-compose up` and verify that no functionality is impacted. If running on an Apple Silicon machine, run `cp docker-compose-arm64.yml docker-compose.override.yml` and then `docker-compose up`. In particular, run any tests to ensure that Concourse runs as expected.
